### PR TITLE
Update ADOPTERS.md

### PR DESCRIPTION
**Pull request by ariel-adam**
Adding Red Hat to the adopters list

### CODE REVIEW

**Comment by ariel-adam** [comment]
@mythi @bpradipt any chance you could review?

**Review by mythi — APPROVED** [review]
nice, LGTM

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ariel-adam** [comment]
@sameo @magowan could you also take a look (seem @mythi was not on the maintainers list :-) ) ?

File: ADOPTERS.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ See list of adopter types at the bottom of this page.
...
 |NanhuLab|Trusted Big Data Sharing System |Beta |Service Provider |The system uses confidential containers to ensure that data users can utilize the data without being able to view the raw data.(No official website yet. For details: yzc@nanhulab.ac.cn) |
+| [Red Hat](https://www.redhat.com/en) | [OpenShift confidential containers](https://www.redhat.com/en/blog/learn-about-confidential-containers) | Beta | Service Provider | Confidential Containers are available from [OpenShift sandboxed containers release version 1.7.0](https://docs.redhat.com/en/documentation/openshift_sandboxed_containers/1.7/) as a tech preview on Azure cloud for both Intel TDX and AMD SEV-SNP. The tech preview also includes support for confidential containers on IBM Z and LinuxONE using Secure Execution for Linux (IBM SEL). Future releases will support bare metal deployments and additional public clouds. |
```
Comment (fitzthum) [inline ADOPTERS.md:16]: Is `IBM SEL` the correct term?

**Review by fitzthum — COMMENTED** [review]
one question

File: ADOPTERS.md (hunk 1 above)
Comment (ariel-adam) [reply ADOPTERS.md:16]: The text is a copy paste from our landing page: https://www.redhat.com/en/blog/learn-about-confidential-containers

So I believe it is

Comment (stevenhorsman) [reply ADOPTERS.md:16]: I've never heard the term `SEL` for it. "IBM Secure Execution for Linux" is the correct name for it and I'm not sure there is an approved abbreviation, so I'd probably just suggest removing that?

**Comment by ariel-adam** [comment]
To simplify things I've removed any mention of IBM-Z since the key platform for RH CoCo is Azure for now.

File: ADOPTERS.md (hunk 1 above)
Comment (stevenhorsman) [reply ADOPTERS.md]: Our Product Manager has stated that SEL is being introduced as an approved abbreviation, so we can leave that as is. Thanks and sorry for any confusion.

**Pull request closed without merging**